examples/spark2/src/main/scala/org/apache/carbondata/examples/HadoopFileExample.scala

```diff
@@ -61,7 +61,7 @@ object HadoopFileExample {
     CarbonInputFormat.setTableName(conf, "Hadoopfile_table")
 
 
-    val input = spark.sparkContext.newAPIHadoopFile(s"${storeLocation}/Hadoopfile_table",
+    val input = spark.sparkContext.newAPIHadoopFile(s"${storeLocation}/hadoopfile_table",
```

Comment:
why you modify only one,this file has serveral ```Hadoopfile_table```?

Reply:
Thanks for your comment @qiuchenjian. I modify this one，then the example go through. I'll check again.

Reply:
> why you modify only one,this file has serveral `Hadoopfile_table`?

I think it's no need to change other `Hadoopfile_table`. When use spark dataframe to save table, it'll use lower case table name as the location, so this pr fix the bug.